tests/Doctrine/Tests/TestUtil.php

```diff
@@ -61,16 +61,28 @@ private static function hasRequiredConnectionParams()
             $GLOBALS['db_type'],
             $GLOBALS['db_username'],
             $GLOBALS['db_password'],
-            $GLOBALS['db_host'],
-            $GLOBALS['db_name'],
-            $GLOBALS['db_port']
+            $GLOBALS['db_name']
+        )
+        && (
+            isset(
+                $GLOBALS['db_host'],
+                $GLOBALS['db_port']
+            ) || isset(
+                $GLOBALS['db_server']
```

Comment:
@deeky666 yes it is like ODBC. I will change this to use the host parameter instead.